xApps/python/lib/e2sm_rc_module.py

```diff
@@ -39,14 +39,25 @@ def _build_ric_control_request(self, control_header, control_msg, ack_request):
         return payload
 
     def send_control_request_style_2_action_6(self, e2_node_id, ue_id, min_prb_ratio, max_prb_ratio, dedicated_prb_ratio, ack_request=1):
-        PLMN = b'00101' # currently not exposed as parameter
-        # S-NSSAI
-        sst = b'1'  # currently not exposed as parameter
-        sd = b'0'   # currently not exposed as parameter
-        # ratios
-        min_prb_policy_ratio = max(0, min(min_prb_ratio, 100))
-        max_prb_policy_ratio = max(0, min(max_prb_ratio, 100))
-        dedicated_prb_policy_ratio = max(0, min(dedicated_prb_ratio, 100))
+        plmn_string = "00101"
+        sst = 1
```

Comment:
can we parameterise these?

Reply:
yes, but currently it is not handled on the gnb side